### DISCUSSION TITLE
How to use TetrahedronCuttingController？

### DISCUSSION THREAD

**Discussion by RickYang1** [InfinyTech3D/SofaUnity#108] [Help - Using SofaUnity] [answered]
Hello，
I try to use `TetrahedronCuttingController` in my simulation，but I encountered some problems，I set the cutting path from `cutPointA` to `cutPointB,` but the actual cutting length is always too long.As follows：
https://github.com/user-attachments/assets/7dd6963e-d849-46f0-a131-d08f6f4da4b8
How can I fix this?
Thanks in advance!

**Comment by epernod** [comment]
Hi,

thank you for the record. I will need to check on my side if I got the same error. Would it be possible to share your scene with the mesh?
By the way, is it a tetrahedral mesh?

Best regards

**Reply by RickYang1** [reply, marked as answer]
> Hi, 你好，
> 
> thank you for the record. I will need to check on my side if I got the same error. Would it be possible to share your scene with the mesh?谢谢你的记录。如果我遇到同样的错误，我需要检查一下。可以与网格共享您的场景吗？ By the way, is it a tetrahedral mesh?顺便问一下，它是四面体网格吗？
> 
> Best regards 此致

Hello,
Thank you for your reply. Unfortunately, the scene has been deleted，the problem I encountered a few months ago, and I suspect it may be due to my mesh being too small, with a diameter of around 0.01m;
Additionally, my mesh is a tetrahedral.
Best wishes！
